### ISSUE TITLE
[Med]: Chrome+Jaws: Screen reader: Visually list looking contents are not marked up as list programatically 

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9464]
## Action Performed:
1. Login with the credentials to Expensify App
2. Click on the green (+) icon located at the bottom of the page.
3. Select Send Money, and a new dialog will appear
4. Navigate and Click to Rupee(currency) symbol link, and a dialog will appear containing various currencies.
5. Navigate to the Currency list and observe the reading announced by Screen Reader.

## Expected Result:
The list of currencies should be announced as a list by a screen reader.

## Actual Result:
When we navigate to the currency list with Screen Reader, the screen reader reads out the currency list as plain text.

HTML:
`<div class="css-1dbjc4n"><div class="css-1dbjc4n"><div class="css-1dbjc4n"><div dir="auto" ...>ALL CURRENCIES</div></div></div>

<div class="css-1dbjc4n"><div class="css-1dbjc4n"><div tabindex="0" class="css-1dbjc4n...>
(...)<div dir="auto" class="css-901oao css-nfaoni...>AED - AED
(...)
<div dir="auto" class="css-901...>CZK - CZK</div>
(...)`

## Workaround:
Yes

## Area issue was found in:
Select a Currency

## Failed WCAG checkpoints
1.3.1

## User impact:
As the content is not marked up as a list, Screen Reader users may face difficulties in understanding the contents and page structure. 

## Suggested resolution:
Markup visually list looking like contents eg: 'AED-AED', 'AEN-AEN', etc..  as list markup.

Refer:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*v1.1.76-5*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

https://user-images.githubusercontent.com/98817464/173807980-7d805714-74a0-47f6-8522-941a71e1eb0c.mp4
